src/components/Header/index.tsx

```diff
@@ -245,8 +247,23 @@ const StyledExternalLink = styled(ExternalLink).attrs({
   }
 `
 
+function getCurrentPageFromLocation(locationPathname: string): PageName {
+  switch (locationPathname) {
+    case '/swap':
+      return PageName.SWAP_PAGE
+    case '/vote':
+      return PageName.VOTE_PAGE
+    case '/pool':
+      return PageName.POOL_PAGE
+    default:
```

Comment:
should this default to unknown or undefined?

Reply:
or even log an error?

Reply:
undefined seems reasonable, error could be misleading i think.